backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -201,6 +201,23 @@ def taxon_preview_link(taxon)
           class: 'btn-outline-secondary', icon: 'view.svg', id: 'admin_preview_taxon', target: :blank
         )
       end
+
+      def select2_local_fallback
+        stripped_locale = I18n.locale.to_s.split('-').first
+        all_select2_locales = %w[
```

Comment:
@MatthewKennedy can we move this into `SELECT2_SUPPORTED_LOCALES` static and freeze it?